### ISSUE TITLE
Consider updating severity levels for MozLog

### ISSUE THREAD

**Issue by ncloudioj** [mozilla-services/Dockerflow#69]
As more of our services have been migrated to GCP, MozLog's severity levels are causing issues as they're different from [those](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity) of GCP Cloud Logging. Dockerflow-compliant services need to remap SLs in order for GCL to render log records properly.

Is it time to update MozLog's SLs to match against GCL now?

@jasonthomas @mostlygeek @jbuck WDYT?

**Comment by mostlygeek** [comment]
@grahamalama is also looking into this, see https://github.com/mozilla-it/ctms-api/issues/771.  

I think it's the right time to have this discussion.  Many things have changed since the mozlog spec was created. 

That said, the severity levels map almost exactly. 

MozLog uses syslog [severity levels](https://en.wikipedia.org/wiki/Syslog#Severity_levels): 
<img width="312" alt="image" src="https://github.com/mozilla-services/Dockerflow/assets/83972/38925760-a677-46ff-9174-ea1ab2d85780">

these are the GCL ones: 

<img width="820" alt="image" src="https://github.com/mozilla-services/Dockerflow/assets/83972/73c12963-1865-40b0-9271-ff016fae1653">

**Comment by ncloudioj** [comment]
Yep, the severity levels match exactly between the two, we just need to update the level values accordingly so that GCL can consume/render them properly.
